OpenTreeMap/otm-core
# Update Rollbar JS client and catch localStorage security error

## Overview

* Updates to the latest version of the Rollbar JS client
* Catches an exception that crashes the map page when a Chrome security setting is enabled while viewing an embedded map.

Connects #3257 
Depends on https://github.com/OpenTreeMap/otm-cloud/pull/460

## Testing Instructions

The tests are Chrome specific 

### Setup

 * Enable third-party cookie blocking chrome://settings/content/cookies
<img width="865" alt="screen shot 2018-10-23 at 10 19 23 am" src="https://user-images.githubusercontent.com/17363/47378710-c7870100-d6ad-11e8-8479-e5bfd91d0c8e.png">

 * Add the hostname of your workstation to `ALLOWED_HOSTS` in `opentreemap/opentreemap/settings/local_settings.py`
 * Restart the app service with `vagrant ssh app -c "sudo service otm-app restart"`
 * Create an `index.html` using the following template, substitute the name of your host, and run a static file server to serve it.
```html
<!doctype html>
<html lang="en">
    <head>
        <meta charset="UTF-8"/>
        <title>Document</title>
    </head>
    <body>
        <iframe src="http://YOUR-HOST-NAME:6060/phillytreemap/map/?embed=1" width="900" height="600"></iframe>
    </body>
</html>
```

### Test


 * Checkout the `develop` branch. 
 * Browse the `index.html` file created in setup and verify that an exception is raised preventing the map from loading
<img width="1444" alt="screen shot 2018-10-23 at 1 46 23 pm" src="https://user-images.githubusercontent.com/17363/47389718-5dc92000-d6ca-11e8-9ca0-8753e4303323.png">
 * Checkout this branch, refresh the embed page, and verify that the error is resolved and the map loads
<img width="1442" alt="screen shot 2018-10-23 at 1 50 06 pm" src="https://user-images.githubusercontent.com/17363/47389877-c44e3e00-d6ca-11e8-8dab-9fc63934f944.png">
